cloud/src/recycler/recycler.cpp

```diff
@@ -985,9 +1099,11 @@ int InstanceRecycler::recycle_indexes() {
         return 0;
     };
 
-    return scan_for_recycle_and_statistics(index_key0, index_key1, "indexes", metrics_context,
-                                           std::move(scan_and_statistics), std::move(recycle_func),
-                                           std::move(loop_done));
+    if (config::enable_recycler_metrics) {
```

Comment:
conf enable_recycler_metrics naming should be more specific 
e.g. `enable_recycler_stats_metrics`